torchtext/experimental/transforms.py

```diff
@@ -251,3 +251,55 @@ def to_ivalue(self):
             vector = self.vector.to_ivalue()
             return VectorTransform(vector)
         return self
+
+
+class PadTransform(nn.Module):
+    r"""Pad sequences. Add padding id to the end of a sequences such that the sequences will
+        have the same length.
+
+    Args:
+        pad_id: the id for padding token
+        return_key_padding_mask: flag if to output padding_mask as output (Default: True)
+
+    Example:
+        >>> pad_id = 2
+        >>> pad_transform = PadTransform(pad_id)
+        >>> seq_batch = [[5, 4, 5, 6, 7], [1, 3], [7, 5, 8]]
+        >>> pad_seq, padding_mask = pad_transform(seq_batch)
+        >>> jit_pad_transform = torch.jit.script(pad_transform)
+    """
```

Comment:
What is the reasoning about adding this new transform, instead of using `torch.nn.utils.rnn.pack_padded_sequence`, which returns a `PackedSequence` object that contains exactly the same information (maybe mashed in a different way)?

Reply:
This is supposed to be easier to use and it's a Module that can be JIT'd